doc/user-guide/indexing.rst

```diff
@@ -536,6 +533,22 @@ __ https://docs.scipy.org/doc/numpy/user/basics.indexing.html#assigning-values-t
       da.isel(x=[0, 1, 2])[1] = -1
       da
 
+You can also assign values in all variables of a ``Dataset`` at once:
+
+.. ipython:: python
+
+    ds = xr.tutorial.open_dataset("eraint_uvz")
+    # by integer
+    ds[dict(longitude=10, latitude=10)] = 1
```

Comment:
If the example is small enough such that it's possible to see the effect of the assignment, could we show the result?

Reply:
+1 it would be great to see an example that is small enough that we could show the result :)